components/CandidateTestView/Instructions/Instructions.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import scss from "../Instructions/Instructions.module.scss";
+import {CandidateTestModel} from "../../../Models/CandidateTestModel";
+
+const InstructionsBlock = (testModel: CandidateTestModel): JSX.Element => (
+    <div>
+        <p className={scss.instructionList}>{testModel.brief}</p>
+        <h2>Example</h2>
+        <p className={scss.instructionList}>{testModel.example}</p>
+        <h2>Input/Output</h2>
+        <ul className={scss.instructionList}>
+            <li className={scss.instructionList}><strong>[execution time limit] {testModel.executionTimeLimit} (js)</strong></li>
+            <li className={scss.instructionList}><strong>[input] {testModel.inputType}</strong><br/>{testModel.inputDescription}</li>
+            <li className={scss.instructionList}><strong>[output] {testModel.outputType}</strong><br/>{testModel.outputDescription}</li>
+        </ul>
+        <h2>[JavaScript (ES6)] Syntax Tips</h2>
+        <p className={scss.instructionList}>{testModel.jsTips}</p>
+        <section className={scss.sampleInputOutput}>
+        <h2>TEST {testModel.number}</h2>
+        <p className={scss.instructionList}><strong>Input:</strong> {testModel.input}</p>
+        <p className={scss.instructionList}><strong>Expected Output:</strong> {testModel.expectedOutput}</p>
```

Comment:
We should also get rid of this - we can't actually run these.

Reply:
What do you mean exactly? :)

Reply:
I think the stuff in the 'tests' section of the question Sasha sent over are the automated tests that Tungl runs against the candidate's code.

We can't run these yet (and won't get that working before the end of the project).

Reply:
Perhaps just input output, as an example?

Reply:
Isn't that what the example is for?

Reply:
ok, removed